spec/license_scout/dependency_manager/glide_spec.rb

```diff
@@ -0,0 +1,137 @@
+#
+# Copyright:: Copyright 2017, Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "license_scout/dependency_manager/glide"
+require "license_scout/overrides"
+require "license_scout/options"
+
+RSpec.describe(LicenseScout::DependencyManager::Glide) do
+
+  subject(:glide) do
+    described_class.new(project_dir, LicenseScout::Options.new(
+      overrides: overrides
+    ))
+  end
+
+  let(:overrides) { LicenseScout::Overrides.new(exclude_default: true) }
+
+  let(:project_dir) { File.join(SPEC_FIXTURES_DIR, "glide") }
+
+  it "has a name" do
+    expect(glide.name).to eq("go_glide")
+  end
+
+  it "has a project directory" do
+    expect(glide.project_dir).to eq(project_dir)
+  end
+
+  describe "when run in a non-glide project dir" do
+    let(:project_dir) { File.join(SPEC_FIXTURES_DIR, "no_dependency_manager") }
+
+    it "does not detect the project" do
+      expect(glide.detected?).to eq(false)
+    end
+  end
+
+  describe "when run in a glide-only project dir" do
+    let(:project_dir) { File.join(SPEC_FIXTURES_DIR, "godep") }
+
+    it "does not detect the project" do
+      expect(glide.detected?).to eq(false)
+    end
+  end
+
+  describe "when run in a glide project dir" do
+    before do
+      ENV["GOPATH"] = File.join(SPEC_FIXTURES_DIR, "godeps_gopath" )
```

Comment:
`godeps` here has nothing to do with the godep dep manager, right? It's just a name...